src/Explorer/Panes/PanelContainerComponent.tsx

```diff
@@ -76,7 +76,44 @@ export class PanelContainerComponent extends React.Component<PanelContainerProps
     );
   }
 
+  public isFocusable(element: HTMLElement) {
```

Comment:
isFocusable and findFocusableParent seem like they might be useful elsewhere? If you think so too, they should be moved to a utility file in a common folder.

Reply:
It is useul for all the instances using this dialog. I will move them to a common folder.